libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -267,7 +267,8 @@ public int hashCode() {
     }
 
     checkArgument(clockRate > 0);
-    return new RtpPayloadFormat(formatBuilder.build(), rtpPayloadType, clockRate, fmtpParameters);
+    return new RtpPayloadFormat(
+        formatBuilder.build(), rtpPayloadType, clockRate, fmtpParameters, mediaEncoding);
```

Comment:
Is `processAacFmtpAttribute` applied to both ADTS AAC (the one we already support) and LATM AAC (this one)?

Reply:
Yes.